source/_posts/2016-09-15-reinventing-our-continuous-learning-practice.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: Reinventing our continuous learning practice
+comments: true
+author:
+  name: Sebastián Martínez
+  email: sebastian@wyeworks.com
+  twitter_handle: smartinez87
+  github_handle:  smartinez87
+  image:  /images/team/sebastian-martinez.jpg
+  description: VP of Engineering at WyeWorks. Ruby on Rails hacker. ExceptionNotification maintainer. Coffee & bacon lover.
+published: true
+---
+
+A bit over a year ago I wrote [this article](https://wyeworks.com/blog/2015/7/16/technical-thursdays-or-how-we-do-continuous-learning/) about continuous learning and
+talked about our Technical Thursdays. If you haven't read it yet, I
+encourage you to do so! In there I talked about the importance of having
+a great team that is curious and passionate about learning.
+
+## Not everything was perfect though
+Some time ago we started to notice a decrease in
+the attendance, people started to skip some sessions. That made us
+question what happened: Did people care less about learning new things?
+Luckly, we were able to see beyond that and realised there were some
+problems with the approach.
+First of all, it was at a fixed time, every
+other Thursday at 9:30am. Let's face it, not everyone *loves* to get
+up that early. It is also known now that people have different times of
+the day where they feel more productive. Another important thing we realised is that
+not everybody has the same interests. Maybe some people were curious
+about a particular technology, but it didn't worth the time for the
+rest.
+
+## Experiment and iterate
+A couple of people of the team saw these issues, and suggested a diferent
+approach as an experiment. They weren't certain it would work, but we
+were all open to give it a shot and test the results after some
+iteration.
+
+The new idea of implementation was simple and awesome at the same time:
+why don't we do a marketplace where people can share the things they
+want to learn, then groups are formed according to what motivates them
+the most, let each group self-manage and coordinate what they'll be
+doing and when, and after that have them present to the rest what they've achieved?
```

Comment:
This is a very long question sentence! I would write it as a regular statement.

I would also remark that we did a transition from a grupal session every 2 weeks to a presentation/workshop session every 2 months (having fewer meetings where the whole team has to agree a time). Otherwise, a big part of the story is missed for the reader, in my opinion.